protocol/migrations/sqlite/1679326850_add_community_token_owners.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE IF NOT EXISTS community_tokens_owners (
+  chain_id INT NOT NULL,
+  address TEXT NOT NULL,
+  owner TEXT NOT NULL COLLATE NOCASE,
+  amount INT NOT NULL,
```

Comment:
Do we not care about tokenID for now?

Reply:
This cache is introduced to help managing communities smart contracts. We need to display list of users and amount of tokens they hold. tokenID will be needed during burn but we can get tokenIds for every user from smart contract during burning operation. Also, in community NFTs, we do not differentiate tokens.